common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -1544,6 +1544,8 @@
   "function.wynntils.stopwatchSeconds.name": "Stopwatch Seconds",
   "function.wynntils.stopwatchZero.description": "Whether the stopwatch is currently at zero",
   "function.wynntils.stopwatchZero.name": "Stopwatch Zero",
+  "function.wynntils.ticks.description": "The number of ticks since joining a world. Resets to 0 at 32-bit int limit.",
```

Comment:
This description is incorrect. The tick count start at 0 when the manager starts, basically at game startup.

To have it reset when joining a world would require you to reset it when joining, and that in turn would make the manager depends on models, which we do not want to do.

Reply:
What does the tickAlways even do compared to onTick then? I meant that the tick counter initially starts on your first world join